var/spack/repos/builtin/packages/discotec/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Discotec(CMakePackage):
+    """This project contains DisCoTec, a code for the distributed sparse
+    grid combination technique with MPI parallelization."""
+
+    homepage = "https://github.com/SGpp/DisCoTec"
+    url = "https://github.com/SGpp/DisCoTec"
+    git = "https://github.com/SGpp/DisCoTec"
+
+    # notify when the package is updated.
+    maintainers = ["freifrauvonbleifrei", "pfluegdk"]
+
+    version("main", branch="main")
+
+    variant("ft", default=False, description="DisCoTec with algorithm-based fault tolerance")
+    variant("gene", default=False, description="Build for GENE (as task library)")
+    variant("hdf5", default=True, description="Interpolation output with HDF5")
+    variant("lto", default=True, description="Build with link-time optimization")
+    variant("openmp", default=True, description="Parallelize with OpenMP")
+    variant("timing", default=True, description="With high-res timers")
+    variant("test", default=True, description="Build Boost tests")
```

Comment:
What is the purpose of this variant? Can we get rid of it and use `self.run_tests` below?

Reply:
it switches whether the tests are built or not -- do you think this would better be part of run_tests?

Reply:
Yes, I think it would be better using `self.run_tests` if it is possible to do so.